src/qt/forms/researcherwizardauthpage.ui

```diff
@@ -39,8 +39,8 @@
        <rect>
         <x>0</x>
         <y>0</y>
-        <width>610</width>
-        <height>460</height>
+        <width>596</width>
+        <height>498</height>
```

Comment:
why these numbers specifically? Why not 600 and 500?

Reply:
You know that when you pull this up in QtCreator and save the form, it changes these automatically...

Reply:
I don't think it is necessary or advisable to try and outguess the layout engine...